posthog/clickhouse/schema.py

```diff
@@ -137,12 +137,14 @@
     KAFKA_SESSION_REPLAY_EVENTS_TABLE_SQL,
     SESSION_REPLAY_EVENTS_TABLE_MV_SQL,
     SESSION_REPLAY_EVENTS_TABLE_SQL,
+    WRITABLE_SESSION_REPLAY_EVENTS_TABLE_SQL,
```

Comment:
this had been failing on everyone's computer for some time already ;-P, cleanups the really annoying exceptions when running tests